pkg/websocket/request_log_messages.go

```diff
@@ -3,6 +3,6 @@ package websocket
 // RequestLogEvent represents incoming request log event messages sent by Stripe.
 type RequestLogEvent struct {
 	EventPayload string `json:"event_payload"`
-	Type         string `json:"type"`
 	RequestLogID string `json:"request_log_id"`
```

Comment:
Could you add a comment to clarify that this is a `resp_` ID and not the same as `EventPayload.RequestID`?